packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -0,0 +1,206 @@
+
+import {View} from "@khanacademy/wonder-blocks-core";
+
+// TODO(scottgrant): Make sure the toolbars are responsive.
+// import type {MediaSize} from "@khanacademy/wonder-blocks-core";
+
+const dismissIcon =
+    "M12.003 10.588L7.707 6.293a1 1 0 0 0-1.414 1.414l4.295 4.296-4.295 4.295a1 1 0 0 0 1.414 1.414l4.296-4.295 4.295 4.295a1 1 0 0 0 1.414-1.414l-4.295-4.295 4.295-4.296a1 1 0 1 0-1.414-1.414l-4.295 4.295z";
+
+type Props = {
+    /**
+     * A list of nodes to render on the left side of the toolbar. In most
+     * cases, this will be empty.
+     */
+    leftContent?: Array<React.Node>,
+
+    /**
+     * The function to call when dismissing the toolbar. Only relevant if
+     * showClose is true.
+     */
+    onClose?: () => void,
+
+    /**
+     * A list of nodes to render on the right side of the toolbar. This will
+     * typically include buttons, links, span elements with text, or nothing
+     * at all.
+     */
+    rightContent?: Array<React.Node>,
+
+    /**
+     * True, if we should show the dismiss icon on the left side o the toolbar.
+     */
+    showClose: boolean,
+
+    /**
+     * How much vertical space to use for the toolbar. If this prop is not
+     * provided, the default is "medium".
+     */
+    size?: "small" | "medium",
+
+    /**
+     * Additional styling for the toolbar. If a color value is provided and
+     * the showClose prop is set to true, the close icon will use this color.
+     */
+    style?: any,
+
+    /**
+     * An optional subtitle rendered in a lighter colour and smaller font size
+     * below the title. Only visible on larger media sizes.
+     */
+    subtitle?: string,
+
+    /**
+     * The main title rendered in larger bold text.
+     */
+    title?: string,
+};
+
+export default class Toolbar extends React.Component<Props> {
+    static defaultProps = {
+        leftContent: [],
+        onClose: () => {},
+        rightContent: [],
+        showClose: false,
+    };
+
+    renderContent(content: React.Node, key: string) {
+        return (
+            <View
+                style={[sharedStyles.content, sharedStyles.verticalAlign]}
+                key={key}
+            >
+                {content}
+            </View>
+        );
+    }
+
+    render() {
+        const {
+            leftContent,
+            onClose,
+            rightContent,
+            showClose,
+            size,
+            style,
+            subtitle,
+            title,
+        } = this.props;
+
+        // TODO(scottgrant): Replace the SVG with an Icon component once
+        // that's done
+        const fillColor = (style && style.color) || "rgba(33, 36, 44, 0.5)";
+        const closeButton = showClose && (
+            <View onClick={onClose}>
+                <svg role="img" width="24px" height="24px">
+                    <path fill={fillColor} d={dismissIcon} />
+                </svg>
+            </View>
+        );
+
+        // TODO(scottgrant): Handle aria-labelledby appropriately.
```

Comment:
Is this just for the close button or something else?

Reply:
This is something we'll need to address with the modals, but I think for now this comment can be removed. I've left an `id` property on the title so it can be referenced using `aria-labelledby` if needed, and since we pass in other children as content items on the left and right, it's possible for parent components to add `id` properties if they're needed.